docs/express-motivation.qmd

```diff
@@ -0,0 +1,343 @@
+---
+title: Motivation
+editor:
+  markdown:
+    wrap: sentence
+format:
+  html:
+    include-in-header: assets/head-content.html
+---
+
+So far, we've really only seen Shiny _Express_: a simple way to learn and create basic apps.
+As we'll discuss here, the same things that make Express simple also make it difficult to manage complexity when your app grows in size and sophistication.[^streamlit-comp]
+And, although we'll cover some techniques for managing complexity and adding sophistication to Express, when you find yourself reaching for these techniques, it might be time to switch to Shiny Core.
```

Comment:
```suggestion
In this article we'll cover some techniques for managing complexity and adding sophistication to Express, but when you find yourself reaching for these techniques too often it might be time to switch to Shiny Core.
```